docs/docsite/rst/dev_guide/developing_collections_creating.rst

```diff
@@ -22,16 +22,18 @@ A user can then install your collection on their systems.
 Creating a collection skeleton
 ==============================
 
-To start a new collection:
+To start a new collection run in your collections directory:
 
 .. code-block:: bash
 
-    collection_dir#> ansible-galaxy collection init my_namespace.my_collection
+    ansible_collections#> ansible-galaxy collection init my_namespace.my_collection
 
 .. note::
 
 	Both the namespace and collection names use the same strict set of requirements. See `Galaxy namespaces <https://galaxy.ansible.com/docs/contributing/namespaces.html#galaxy-namespaces>`_ on the Galaxy docsite for those requirements.
 
+It will create the structure ``[my_namespace]/[my_collection]/[collection skeleton]``
+The git project must be initialized in the collection directory.
```

Comment:
There is no obligation to use Git or to version-control the source code at all. So using "must" seems misleading at best. Not sure if Git even deserves to be mentioned in such a context. Maybe some `.. hint::` admonition could be useful, though.

Reply:
Agree, changed.